code/lib/preview-api/src/modules/preview-web/PreviewWeb.test.ts

```diff
@@ -180,6 +182,15 @@ describe('PreviewWeb', () => {
       expect(preview.storyStore.globals!.get()).toEqual({ a: 'c' });
     });
 
+    it('emits the PROJECT_PREPARED event', async () => {
```

Comment:
the title of all of these new tests are all very similar. They should describe their own individual cases so they are easier distinguishable. I guess they all need a "when ..."

Reply:
@JReinhold isn't that what the outer describe is for? This one is:

<img width="541" alt="image" src="https://user-images.githubusercontent.com/132554/232015172-33df4a20-5fa1-4677-a187-7fdccae96d91.png">

Reply:
You're right, I missed that part.